src/components/panel/panel.js

```diff
@@ -2087,6 +2087,9 @@ function MdPanelPosition($injector) {
   /** @private {boolean} */
   this._isRTL = $injector.get('$mdUtil').bidi() === 'rtl';
 
+  /** @private @const {!angular.$mdConstant} */
```

Comment:
I'm not sure. I found it being used this way in [another place in mdPanel](https://github.com/angular/material/blob/master/src/components/panel/panel.js#L1051).